simple_https_client/src/tls13client.rs

```diff
@@ -15,45 +44,28 @@ fn main() -> anyhow::Result<()> {
     // Setup tracing.
     tracing_subscriber::fmt::init();
 
-    // Obtain host and port from arguments.
-    let (host, port) = {
-        let mut args = env::args();
+    let flags = flags::Connect::from_env_or_exit();
 
-        let _ = args.next().context("Unexpected parameter environment.")?;
-
-        let host = args.next().unwrap_or_else(|| "www.google.com".to_string());
-        let port = args.next().unwrap_or_else(|| "443".to_string());
-
-        (
-            host,
-            u16::from_str(&port).context("Failed to parse port number.")?,
-        )
+    let (host, port, algorithms) = {
+        let host = flags.host.unwrap_or("www.google.com".to_string());
+        let port = flags.port.unwrap_or(443);
+        let algorithms = flags.algorithms.map(|s| ciphersuite_from_str(&s).unwrap());
+        (host, port, algorithms)
     };
 
     let request = format!("GET / HTTP/1.1\r\nHost: {}\r\n\r\n", host);
 
-    // FIXME: #51 This is going to go away as soon as Bertie supports multiple
-    //        ciphersuites.
-    let mut response_prefix = Vec::new();
-    for algorithms in ciphersuites() {
-        // Initiate HTTPS connection to host:port.
-        let stream = TcpStream::connect((host.clone(), port))?;
-        stream.set_nodelay(true).expect("set_nodelay call failed");
-        trace!(
-            host = format!("{}:{}", host, port),
-            "Opened TCP connection to host."
-        );
+    // Initiate HTTPS connection to host:port.
```

Comment:
Is #51 fixed with this? Please document what this PR is doing there.

Reply:
I would say #51 implies that the client would offer / try multiple suites and the server could handle multiple suites without restart? In that case this PR does not fully address the issue. (I've commented there as well.)